src/core/MOM_dynamics_split_RK2.F90

```diff
@@ -963,6 +1007,25 @@ subroutine step_MOM_dyn_split_RK2(u, v, h, tv, visc, Time_local, dt, forces, p_s
   !  enddo ; enddo ; enddo
   !  call post_data(CS%id_hf_v_BT_accel, hf_v_BT_accel, CS%diag)
   !endif
+  if (CS%id_intz_u_BT_accel_2d > 0) then
+    allocate(intz_u_BT_accel_2d(G%IsdB:G%IedB,G%jsd:G%jed))
```

Comment:
These shouldn't be allocated and deallocated each and every timestep

Reply:
Thanks for pointing this out. I will the necessary changes.